radicle-httpd/src/lib.rs

```diff
@@ -95,8 +88,56 @@ pub async fn run(options: Options) -> anyhow::Result<()> {
         )
         .into_make_service_with_connect_info::<SocketAddr>();
 
-    axum::Server::bind(&options.listen)
+    axum::Server::bind(&listen)
         .serve(app)
         .await
         .map_err(anyhow::Error::from)
 }
+
+/// Create a router consisting of other sub-routers.
+fn router(options: Options, profile: Profile) -> anyhow::Result<Router> {
+    tracing::info!("using radicle home at {}", profile.home().display());
```

Comment:
I think this log line should be in the `run` function.

Reply:
yup, my thoughts too. https://github.com/radicle-dev/heartwood/pull/422/commits/75463a7872255a89490b362f7e11962c56cd15f2#diff-f5911c8765b18e1a9d9d0957001f22ff11cc9c0f5749f9293fae412ccd681362R56